definitions/variable/emissions/emissions.yaml

```diff
@@ -43,52 +71,113 @@
     description: Emissions of {Level-3 Species} from soil management practices
       in the agriculture sector
     unit: "{Level-3 Species}"
+- Emissions|{Level-3 Species}|AFOLU|Biomass Burning:
+    description: Emissions from agricultural waste burning
+    unit: "{Level-3 Species}"
 - Emissions|{Level-3 Species}|AFOLU|Land:
     description: Emissions of {Level-3 Species} from forestry and other land use
     unit: "{Level-3 Species}"
+    components:
+      - Emissions|{Level-3 Species}|AFOLU|Land|Forest Land
+      - Emissions|{Level-3 Species}|AFOLU|Land|Cropland
+      - Emissions|{Level-3 Species}|AFOLU|Land|Grassland
+      - Emissions|{Level-3 Species}|AFOLU|Land|Wetlands
+      - Emissions|{Level-3 Species}|AFOLU|Land|Settlements
+      - Emissions|{Level-3 Species}|AFOLU|Land|Other Land
+      - Emissions|{Level-3 Species}|AFOLU|Land|Fires
+- Emissions|{Level-3 Species}|AFOLU|Land|Forest Land:
+    description: Emissions and removals from lands with woody vegetation
+      (partially IPCC 1996 category 5A/5B/5D; IPCC 2006 category 3B1) 
+    note: It also includes systems with vegetation that currently fall below, 
+      but are expected to later exceed, the threshold values used by a country to
+      define the forest land category.
+    unit: "{Level-3 Species}"
+- Emissions|{Level-3 Species}|AFOLU|Land|Cropland:
+    description: Emissions and removals from arable and tillage land, rice
+      fields, and agro-forestry systems where vegetation falls
+      below the thresholds used for the forest land category
+      (no clear IPCC 1996 category; IPCC 2006 category 3B2)
+    note: This does not include methane emissions from rice cultivation
+    unit: "{Level-3 Species}"
+- Emissions|{Level-3 Species}|AFOLU|Land|Grassland:
+    description: Emissions and removals from rangelands and pasture land
+      that is not considered cropland. It also includes systems with
+      woody vegetation that fall below the threshold values used
+      in the forest land category and are not expected to exceed
+      them, without human intervention (partially IPCC 1996 category 4D/4E/5A/5B/5C/5D; IPCC 2006 category 3B3).
+    note: The category also includes all grassland from wild lands to recreational 
+      areas as well as agricultural and silvi-pastural systems, subdivided
+      into managed and unmanaged, consistent with national
+      definitions.
+    unit: "{Level-3 Species}"
 - Emissions|{Level-3 Species}|AFOLU|Land|Wetlands:
-    description: Emissions of {Level-3 Species} from managed peatlands (drained and rewetted)
+    description: Emissions from land that is covered or saturated by water for
```

Comment:
This would be only GHG emissions from rewetted peatlands / wetlands?
Because GHG emissions intact peatlands / wetlands are not antrophogenic.
Or should all GHG emissions from drained and rewetted peatands be reported in this category?